src/WebJobs.Extensions.DurableTask/OutOfProcMiddleware.cs

```diff
@@ -540,6 +581,23 @@ public async Task CallActivityAsync(DispatchMiddlewareContext dispatchContext, F
             dispatchContext.SetProperty(activityResult);
         }
 
+        private static DurableHttpRequest? ConvertDurableHttpRequest(string? inputString)
+        {
+            IList<dynamic>? input = JsonConvert.DeserializeObject<IList<dynamic>>(inputString);
```

Comment:
I'm wondering if there's a better way to deserialize the `DurableHttpRequest` here since using `dynamic` might not be reliable. I ended up using this since `DurableHttpRequest` is serialized using the System.Text.Json serializer in the worker extension, but deserialized using the Newtonsoft.Json serializer in the webjobs extension, which leads to different json payloads. Let me know if there's a recommended way to go about this.

Reply:
I recommend writing custom converter for STJ so that it will produce identical serialization payload to what we expect for newtonsoft, then you can serialize from STJ and deserialize directly into Newtonsoft.